staging/src/k8s.io/kube-apiextensions-server/pkg/controller/status/naming_controller.go

```diff
@@ -230,12 +231,20 @@ func equalToAcceptedOrFresh(requestedName, acceptedName string, usedNames sets.S
 func (c *NamingConditionController) sync(key string) error {
 	inCustomResourceDefinition, err := c.crdLister.Get(key)
 	if apierrors.IsNotFound(err) {
+		if err := c.requeueAllOtherGroupCRDs(key); err != nil {
```

Comment:
deserves a comment: CRD was deleted and has freed its names. Reconsider all other CRDs in the same group.